Jenkinsfile

```diff
@@ -1011,17 +1011,20 @@ pipeline {
                     }
                     post {
                         always {
-                            junit testResults: 'bundles/junit-report-*.xml', allowEmptyResults: true
+                            junit testResults: 'bundles/junit-report-*.xml', allowEmptyResults: false
                             catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE', message: 'Failed to create bundles.tar.gz') {
                                 powershell '''
+                                $ErrorActionPreference = 'Stop'
+                                cd $env:WORKSPACE
```

Comment:
interesting; is Workspace not the default directory in which Jenkins runs? (mostly curiousity from my side; not super familiar with everything Jenkins ☺️)

Reply:
Afaiu, it is on `stages{}` but now on `post{}` phase.

Reply:
Ah, that would explain. Sounds like I need to do some reading up on that, thanks!

(was wondering if we could simplify further)